### ISSUE TITLE
Linux Debian exec failure

### ISSUE THREAD

**Issue by michail-vestnik** [grain-lang/grain#1362]
Hi!

The official site has a link to download the Grain binary.
The problem is that Debian will not start the binary if it is downloaded from the internet or USB.
Please distribute the file in zip, tar.gz or rar archives, Debian and more distr. can only work with these files.

Thanks!

**Comment by sanathusk** [comment]
Can you explain in more detail , what exact steps did follow, when the installer failed?
i was able to install grain successfully on Ubuntu 22.024 (Debian based distrubution ) by running the curl script mention in the [docs](https://grain-lang.org/docs/getting_grain#Linux-x64---Download)
```
sudo curl -L --output /usr/local/bin/grain \
https://github.com/grain-lang/grain/releases/download/grain-v0.5.2/grain-linux-x64 \
&& sudo chmod +x /usr/local/bin/grain
```
If you have just downloaded the binary without using curl, it should be still possible to install in using following command:

``` 
cd dir_that_has_binary 
sudo cp grain-linux-x64 /usr/local/bin/grain && sudo chmod +x /usr/local/bin/grain

**Comment by michail-vestnik** [comment]
@sanathusk I wanted to download Grain binary and transfer it to another computer where there is no internet.
The "download it" link contains a binary that does not run in Debian. 

![x](https://user-images.githubusercontent.com/87961969/179893306-9b2c44f8-a201-43dc-a914-2713f746f8d7.png)

If you use curl, there is no problem.
Maybe the problem is on my side but I checked on two different computers and decided to warn you.
Now I copied the installed Grain from the usr/local/bin folder and moved it to another computer the problem is solved.
Thank you.
